tests/phpunit/unit-tests/Model/Test_Model_System_Report.php

```diff
@@ -1,4 +1,5 @@
 <?php
+declare( strict_types=1 );
```

Comment:
Sorry, these declarations need to go in the non-test MVC classes. But you can leave them in the Test files too.